app/models/manager_refresh/inventory/automation_manager.rb

```diff
@@ -3,6 +3,16 @@ module ManagerRefresh::Inventory::AutomationManager
   include ::ManagerRefresh::Inventory::Core
 
   class_methods do
+    def has_automation_manager_configuration_workflows(options = {})
```

Comment:
Can these be renamed as Rubocop suggested?

Reply:
there are others, so I am not sure if we want to do this. Probably not this PR, even if we want to.

Reply:
ok, done

Reply:
Hey @Ladas thinking about the naming about these `has_*`, are there meant to be a predicate? or they should be `add_*`?
That could be confusing rubocop...

Reply:
Yeah, it's not a question (rubocop is wrong), so let's keep this consistent. FYI @slemrmartin is redoing the interface, so it will not look like this at all